internal/setup/setup_test.go

```diff
@@ -791,6 +811,327 @@ var setupTests = []setupTest{{
 			},
 		},
 	},
+}, {
+	summary: "Archives with keyrings",
+	input: map[string]string{
+		"chisel.yaml": `
+			format: chisel-v1
+			archives:
+				foo:
+					version: 22.04
+					components: [main, universe]
+					suites: [jammy]
+					public-keys: [ubuntu-archive]
+					default: true
+				bar:
+					version: 22.04
+					components: [universe]
+					suites: [jammy-updates]
+					public-keys: [ubuntu-archive, ubuntu-master]
+			public-keys:
+				ubuntu-archive: |
+					-----BEGIN PGP PUBLIC KEY BLOCK-----
+					
+					mQINBE+tgXgBEADfiL1KNFHT4H4Dw0OR9LemR8ebsFl+b9E44IpGhgWYDufj0gaM
```

Comment:
It's unfortunate to have a couple hundred lines of stuff in the test output. Can we just assume that the embedded files are correct here, and do something like this:

```go
input = `
format: chisel-v1
...
public-keys:
  ubuntu-archive: |
` + indentLines(testKeyringUbuntuArchiveASCII, "\t\t\t\t\t")  + `
  ubuntu-master: |
` + indentLines(testKeyringUbuntuMasterASCII, "\t\t\t\t\t") + `
...
`
```

Will need a small helper function to do the indentation, but that seems a lot cleaner than including all these lines in the `test.go` file.

Reply:
Indeed. Did just as you suggested, except I kept it on the same line with `+ "\n" +`. There's now extra new line between map entries, but I don't think it matters that much.